exec/java-exec/src/main/java/org/apache/drill/exec/server/options/SystemOptionManager.java

```diff
@@ -70,11 +69,13 @@
  *  </p>
  */
 
-public class SystemOptionManager extends BaseOptionManager implements OptionManager, AutoCloseable {
+public class SystemOptionManager extends BaseOptionManager implements AutoCloseable {
   private static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(SystemOptionManager.class);
 
-  private CaseInsensitiveMap<OptionValidator> VALIDATORS;
-  public void populateValidators() {
+  public static final CaseInsensitiveMap<OptionValidator> DEFAULT_VALIDATORS =
```

Comment:
I wonder, was this an intentional regression, or has your code not been rebased onto the latest master (with Jythosna's changes) and so your commit reintroduces old code?

Reply:
Not a regression see comment above.